pages.it/linux/uname.md

```diff
@@ -0,0 +1,36 @@
+# uname
+
+> Uname stampa informazioni a proposito della macchina e del sistema operativo in esecuzione su di essa
+> Maggiori informazioni: <https://www.gnu.org/software/coreutils/manual/html_node/uname-invocation.html>.
+
+- Stampa tutte le informazioni:
+
+`uname --all`
+
+- Stampa il nome corrente del kernel:
+
+`uname --kernel-name`
+
+- Stampa il nome host corrente del nodo di rete:
+
+`uname --nodename`
+
+- Stampa il rilascio del kernel corrente:
```

Comment:
```suggestion
- Stampa la release del kernel corrente:
```
Literally translating `release` here does not make much sense, it's a technical term that can be kept as is.